### ISSUE TITLE
What is the function of mirror: true

### ISSUE THREAD

**Issue by wushu037** [michalsnik/aos#588]
I didn't figure out what it was for
It's not in the documentation

**Comment by r0b3r70** [comment]
It's supposed to fade out as you scroll though. It's in the readme. Doesn't seem to work though :/

**Comment by Antimated** [comment]
> It's supposed to fade out as you scroll though. It's in the readme. Doesn't seem to work though :/

Yeah setting "mirror" doesn't seem to do anything, at least not programmatically :(
